docs-markdown/src/controllers/cleanup/removeEmptyMetadata.ts

```diff
@@ -37,8 +37,10 @@ export function removeEmptyMetadata(progress: any, file: string, files: string[]
 }
 
 export function deleteEmptyMetadata(data: any) {
-    const metadataRegex: any = new RegExp(/^(\w+\.*\w+?:)(\s*|\s""|\s'')[\n|\r](?=(.|\n|\r)*---\s$)/gmi);
-    data = data.replace(metadataRegex, "");
+    const metadataListRegex: any = new RegExp(/^(\s+\-)(\s*|\s""|\s'')[\n|\r](?=(.|\n|\r)*---\s$)/gmi);
+    data = data.replace(metadataListRegex, "");
+    const metadataRegex: any = new RegExp(/^(\w+\.*\w+?:)(\s*|\s""|\s'')(?!\n*\s+\-\ (\s*|\s""|\s''))[\n|\r](?=(.|\n|\r)*---\s$)/gmi);
+    data = data.replace(metadataRegex, "")
```

Comment:
I know semi-colons are optional in javascript, and I know this is just a nitpick, but please add a semi-colon here.

Reply:
My mistake, I will fix this!